tests/test_cmdline.py

```diff
@@ -82,3 +83,26 @@ def test_handle_value_errors(mocked_TNEF):
     with pytest.raises(SystemExit):
         from tnefparse.cmdline import tnefparse
         tnefparse()
+
+
+def test_cmdline_logging_info(caplog, capsys):
+    retv = tnefparse.cmdline.tnefparse(('-l', 'INFO', '-rb', 'tests/examples/rtf.tnef'))
+    assert not retv
+    stdout, _ = capsys.readouterr()
+    assert len(stdout) == 593
+    assert caplog.record_tuples == [
+        ('tnefparse', logging.INFO, 'Skipping checksum for performance'),
+    ]
+
+
+def test_cmdline_logging_warn(caplog, capsys):
+    retv = tnefparse.cmdline.tnefparse(('-l', 'WARN', '-rb', 'tests/examples/rtf.tnef'))
+    assert not retv
+    stdout, _ = capsys.readouterr()
+    assert len(stdout) == 593
+    assert caplog.record_tuples == []
+
+
+def test_cmdline_logging_illegal():
+    with pytest.raises(SystemExit):
+        tnefparse.cmdline.tnefparse(('-l', 'ILLEGAL', '-rb', 'tests/examples/rtf.tnef'))
```

Comment:
The tests look great. Thanks for adding them!

Could you please add one more test which proves that without passing a logging option that the logger still is at error level?